pkg/sql/pgwire/types.go

```diff
@@ -369,7 +369,8 @@ func (b *writeBuffer) writeBinaryDatum(
 		subWriter.putInt32(int32(hasNulls))
 		subWriter.putInt32(int32(v.ParamTyp.Oid()))
 		subWriter.putInt32(int32(v.Len()))
-		subWriter.putInt32(int32(v.Len()))
+		// Lower bound, we only support a lower bound of 1.
+		subWriter.putInt32(1)
```

Comment:
I'm curious how you caught this. What was failing?

Reply:
The tests I just added were, haha. But I guess this is a good indicator that nobody really makes use of the lower bound.